components/Calendar.js

```diff
@@ -156,63 +155,81 @@ export default class Calendar extends Component {
     }
   }
 
-  renderMonthView(argMoment, eventsMap) {
+  getStartMoment(calFormat, currMoment) {
+    const weekStart = this.props.weekStart;
+
+    let r;
```

Comment:
This is "borrowed" from the C convention. "r" is the return value, but I'll comply to you request.